editor/src/clj/editor/console.clj

```diff
@@ -413,26 +424,27 @@
                              (data/->Cursor row (count line)))
     :type type))
 
-(defn- make-line-regions [resource-map on-region-click! ^long row [type line]]
+;; todo fit in 4 args?
+(defn- make-line-regions [resource-map resource-suffix-map-delay on-region-click! row [type line]]
```

Comment:
I had to remove the type-hint since they are supported only up to 4 args (i.e. it was no longer compiling)

Reply:
Agreed, I'll remove the todo.